Government/Estate fixing

Brother-Zael opened the pull request:
Fixed localization for statists mechanics regarding the settled khanate and romance senate. Fixed Noble estate appearing for Settled Khanate reform when it shouldn't. In order to do the latter I needed to include the noble estate file and the estate folder. Figured I'd include all of the estate folder to avoid any issues and so it's that much easier to mod in the future.

Brother-Zael committed:
Government/Estate fixing

Fixed localization for statists mechanics regarding the settled khanate and romance senate. Fixed Noble estate appearing for Settled Khanate reform when it shouldn't. In order to do the latter I needed to include the noble estate file and the estate folder. Figured I'd include all of the estate folder to avoid any issues and so it's that much easier to mod in the future.

PitiRR commented:
> Figured I'd include all of the estate folder to avoid any issues and so it's that much easier to mod in the future.

No, please no

Brother-Zael committed:
Remove excess estate files

Removed Excess Estate Files because pitirr was really adamant about keeping it simple, stupid

Brother-Zael committed:
Added Romanian gov't, scotland privilege

Added the Romanian unique gov't, "Romance Empire" and the unique privilege from Scotland rp event

PitiRR reviewed (approved):
lgtm
